src/Tablero.cpp

```diff
@@ -241,4 +241,13 @@ bool Tablero::insertarFicha(Color ficha, int columna) {
   return false;
   // se retorna falso, para que la GUI sepa que no se logró insertar la ficha
   // en esa columna y deje que el usuario haga otro click.
+}
+
+bool Tablero::empate() {
```

Comment:
el método lo que hace es que busca la primera fila de todas las columnas. En cuanto encuentra un 0, que es non-color, devuelve falso, porque significa que aún hay al menos un campo para meterle alguna otra ficha. Si logra pasar por todo el for y aún no se ha devuelto, entonces devuelve verdadero, ya que efectivamente el tablero está lleno y si aún no hay ganador, hay empate.